src/gui/folder.h

```diff
@@ -365,6 +378,21 @@ private slots:
      * path.
      */
     bool _saveBackwardsCompatible;
+
+    /**
+     * Watches this folder's local directory for changes.
+     *
+     * Created by registerFolderWatcher(), triggers slotWatchedPathChanged()
+     */
+    FolderWatcher *_folderWatcher = 0;
```

Comment:
ohoh!  you are putting the FolderWatcher back in the Folder itself :-)  cc/ @dragotin